src/ploigos_step_runner/step_implementers/sign_container_image/curl_push.py

```diff
@@ -168,10 +173,14 @@ def __curl_file(
         container_image_signature_url = f"{signature_server_url}/{container_image_signature_name}"
 
         # calculate hashes
-        with open(container_image_signature_file_path, 'rb') as container_image_signature_file:
-            container_image_signature_file_contents = container_image_signature_file.read()
-            signature_file_md5 = hashlib.md5(container_image_signature_file_contents).hexdigest()
-            signature_file_sha1 = hashlib.sha1(container_image_signature_file_contents).hexdigest()
+        if not with_fips:
+            with open(container_image_signature_file_path, 'rb') as container_image_signature_file:
+                container_image_signature_file_contents = container_image_signature_file.read()
+                signature_file_md5 = hashlib.md5(container_image_signature_file_contents).hexdigest()
+                signature_file_sha1 = hashlib.sha1(container_image_signature_file_contents).hexdigest()
+        else:
+            signature_file_md5 = None
```

Comment:
Not sure what the point of clearing these values out as it's not used if fips.

Reply:
clearing the values prevents an error: `UnboundLocalError: local variable 'signature_file_md5' referenced before assignment`

Reply:
I guess my comment was more of a stylistic nitpick.  

1. If you create `curl_additional_options = []` earlier in the method, you can simply add the values in the block starting on line 176 and save you the additional `if` on line 193.  

2. Due to the return at the end of the method, you could do like you did on line 192 and initialize the two necessary variables (`signature_file_sha1`, and `signature_file_md5`) to `None` somewhere near the top of this method.

But like I said, minor issue.

Reply:
Thanks for the comments! I did what you suggested and attempted to merge my code style with the existing style.